CBC/client-assets/assets/configurations/feature/apps.json

```diff
@@ -287,6 +286,110 @@
       "toolTip": "Author",
       "shortName": "Author",
       "url": "/author/home"
+    }, "feature_cbp": {
+      "accessKey": "",
+      "icon": "/assets/icons/CBP_portal.svg",
+      "iconType": "mat-icon",
+      "id": "feature_cbp",
+      "keywords": [
+        "author",
+        "authoring tool"
+      ],
+      "isExternal": "true",
+      "name": "CBP",
+      "permission": [
+        "editor",
+        "admin",
+        "content_publisher",
+        "content_creator",
+        "content-creator",
+        "content_reviewer"
+      ],
+      "show": true,
+      "status": "live",
+      "toolTip": "Author",
+      "shortName": "Author",
+      "url": "https://cbp.igot-dev.in/"
+    },
+    "feature_cbc": {
+      "accessKey": "",
+      "icon": "/assets/icons/CBC_portal.svg",
+      "iconType": "mat-icon",
+      "id": "feature_cbc",
+      "keywords": [
+        "cbc"
+      ],
+      "isExternal": "true",
+      "name": "CBC",
+      "permission": [
+        "cbc_admin",
+        "CBC_MEMBER"
+      ],
+      "show": true,
+      "status": "live",
+      "toolTip": "Author",
+      "shortName": "Author",
+      "url": "https://cbc.igot-dev.in/"
+    },
+    "feature_mdo": {
+      "accessKey": "",
+      "icon": "/assets/icons/MDO_portal.svg",
+      "iconType": "mat-icon",
+      "id": "feature_mdo",
+      "keywords": [
+        "mdo"
+      ],
+      "isExternal": "true",
+      "name": "MDO",
+      "permission": [
+        "admin",
+        "mdo_admin",
+        "wat_user",
+        "wat_member"
+      ],
+      "show": true,
+      "status": "live",
+      "toolTip": "MDO",
+      "shortName": "MDP",
+      "url": "https://mdo.igot-dev.in/"
+    },
+    "feature_spv": {
+      "accessKey": "",
+      "icon": "/assets/icons/SPV_portal.svg",
+      "iconType": "mat-icon",
+      "id": "feature_spv",
+      "keywords": [
+        "spv"
+      ],
+      "isExternal": "true",
+      "name": "SPV",
+      "permission": [
+        "superAdmin",
+        "spv_admin"
+      ],
+      "show": true,
+      "status": "live",
+      "toolTip": "SPV",
+      "shortName": "SPV",
+      "url": "https://spv.igot-dev.in/"
```

Comment:
This url is incorrect.. 
For SPV url is https://admin-sphere.aastrika.org/

Reply:
Do review now